### PR TITLE
YAPF pep8

### PR DESCRIPTION
**Pull request by berggren**
Use vanilla pep8, except keeping 2 spaces indent.

### CODE REVIEW

File: turbinia/pubsub.py [hunk 1]
```diff
@@ -56,14 +59,15 @@ def to_json(self):
       A JSON serialized object.
     """
     serializable = copy.deepcopy(self.__dict__)
-    serializable['evidence'] = [x.serialize() for x in serializable['evidence']]
```
Comment (aarontp) [inline turbinia/pubsub.py]: Why wrap this??

File: tools/plaso_pubsub.py [hunk 2]
```diff
@@ -279,7 +280,8 @@ def _CopyFileToBucket(self, client, filename, object_name):
       error_count = 0
       try:
         status, response = request.next_chunk()
-      except (google_errors.HttpError, google_errors.ResumableUploadError) as e:
+      except (google_errors.HttpError,
+              google_errors.ResumableUploadError) as e:
```
Comment (aarontp) [inline tools/plaso_pubsub.py]: Why wrap here?

File: turbinia/processors/google_cloud.py [hunk 3]
```diff
@@ -396,8 +399,8 @@ def PostprocessDetachDisk(evidence):
 
   config.LoadConfig()
   instance_name = GetLocalInstanceName()
-  project = GoogleCloudProject(project_id=config.PROJECT,
-                               default_zone=config.ZONE)
+  project = GoogleCloudProject(
+      project_id=config.PROJECT, default_zone=config.ZONE)
```
Comment (aarontp) [inline turbinia/processors/google_cloud.py]: I don't get this one either.  Everywhere else it seems that it likes to put the first parameter on the same line (like this was previously), and then put the second parameter on the second line even with it.  Why does it handle it differently here?

**Comment by berggren** [comment]
Dropping this one, let's fix up yapf at a later time

**Pull request closed without merging**